src/main/scala/api/MyMap.scala

```diff
@@ -0,0 +1,98 @@
+package api
+
+
+import scala.collection.JavaConverters._
+
+import java.util.concurrent.ConcurrentHashMap
+
+import javax.servlet.http.{HttpServlet, HttpServletRequest, HttpServletResponse}
+
+import scala.collection.concurrent
+
+object MyMap {
+  val map: concurrent.Map[String, String] = new ConcurrentHashMap[String, String].asScala
+
+  class getServlet extends HttpServlet {
+    override protected def doGet(req: HttpServletRequest, resp: HttpServletResponse): Unit = {
+      var key = req.getParameter("key")
```

Comment:
Use `var`s only when absolutely needed. Immutability is better, use `val` whenever possible.